feature/bgp/admin_distance/README.md

```diff
@@ -0,0 +1,146 @@
+*   Validate that ```ipv4-network-1 = 192.168.10.0/24``` is NOT a valid route through the eBGP session on port-1 and invalid route reason is ```BGP_NOT_SELECTED_BESTPATH```
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/loc-rib/routes/route/state/valid-route
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/loc-rib/routes/route/state/invalid-reason
+*   Ensure that ```ipv6-network-1 = 2024:db8:64:64::/64``` is received through both the eBGP session and the IS-IS session on port-1
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/prefix
+    *   /network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/ipv6-reachability/prefixes/prefix
+*   Validate that ```ipv6-network-1 = 2024:db8:64:64::/64``` is NOT a valid route through the eBGP session on port-1 and invalid route reason is ```BGP_NOT_SELECTED_BESTPATH```
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/state/valid-route
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/state/invalid-reason
+
+*   Configure Administrative Distance or Preference of iBGP session on port-2 to 40
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/default-route-distance/config/internal-route-distance
+*   Ensure that ```ipv4-network-2 = 192.168.20.0/24``` is received through both the iBGP session and the IS-IS session on port-2
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/loc-rib/routes/route/prefix
+    *   /network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/ipv4-internal-reachability/prefixes/prefix
+*   Validate that ```ipv4-network-2 = 192.168.20.0/24``` is a valid route through the iBGP session on port-2
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv4-unicast/loc-rib/routes/route/state/valid-route
+*   Ensure that ```ipv6-network-2 = 2024:db8:128:128::/64``` is received through both the iBGP session and the IS-IS session on port-2
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/prefix
+    *   /network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/tlvs/tlv/ipv6-reachability/prefixes/prefix
+*   Validate that ```ipv6-network-2 = 2024:db8:128:128::/64``` is a valid route through the iBGP session on port-2
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/state/valid-route
```

Comment:
Recommend changing test design to use a 3 port design and use route-preference to change which port traffic is forwarded on.  IMO, this simplifies the test to configure route-distance and measure the impact via traffic forwarding to the ATE.  Requiring OC paths for accessing the DUT BGP RIB's and ISIS link state database paths should be out of scope of this test IMO.

Reply:
Agreed, I will use a 3 port design now to validate route-distance. Will remove the unnecessary validations